datalad/metadata/search.py

```diff
@@ -320,7 +325,8 @@ def pretty_str(s):
                            for k, v in s.items()])
     elif isinstance(s, text_type):
         try:
-            return s.encode('utf-8')
+            b = s.encode('utf-8')
+            return (os.linesep + "  ").join(_lines_regex.split(b))
```

Comment:
I am in favor of that, but didn't have the brilliant idea yet to do all in one go.